src/NzbDrone.Core/Download/Clients/Nzbget/NzbgetSettings.cs

```diff
@@ -55,6 +55,9 @@ public NzbgetSettings()
         [FieldDefinition(7, Label = "Use SSL", Type = FieldType.Checkbox)]
         public bool UseSsl { get; set; }
 
+        [FieldDefinition(8, Label = "Add Paused", Type = FieldType.Checkbox)]
```

Comment:
Looks like this requires a specific NZBGet version, if so `HelpText` should be added to indicate that.